omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -133,6 +133,18 @@ def import_legacy_service_config(old_service_key, new_service_key, keys)
       end
     end
 
+    def deprecated_postgresql_settings
```

Comment:
Could have also gone into [the postgres preflight validator](https://github.com/chef/chef-server/blob/366438de00fc294798441f0a5cba440dc52a2b42/omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb#L95-L103), I suppose? If you've avoided that for a reason, we might want to migrate the other check here, too? I can see that the exact message is different, but I wonder about the place where these kinds of checks should happen.